e2e/e2eApi.ts

```diff
@@ -39,7 +39,22 @@ export class E2EApi {
     return (await this.count(selector)) > 0
   }
 
-  public clickOn = async (selector: string) => await (await this.getElement(selector)).click()
+  public clickOn = async (selector: string, x?: number, y?: number) => {
+    if (x === undefined && y === undefined) {
+      const elementHandle = await this.getElement(selector)
+      await elementHandle.click()
+
+      return
+    }
+
+    const dimensions: { x: number; y: number } = await this.page.evaluate(s => {
```

Comment:
`boundingbox` method of `ElementHandle` can be used here: https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#elementhandleboundingbox